.github/workflows/build-cmake.yaml

```diff
@@ -0,0 +1,176 @@
+            cxx: clang++
+            developer-dir: /Applications/Xcode_16.app/Contents/Developer
+        exclude:
+          - environment: { no-unity: ON }
+            cmake-build-type: Release
+
+    name: "${{ matrix.environment.os }} ${{ matrix.environment.cc }} ${{ matrix.cmake-build-type }}${{ matrix.environment.no-unity == 'ON' && ' No Unity' || '' }}"
+
+    runs-on: ${{ matrix.environment.os }}
+
+    env:
+      BUILD_DIR: build
+      DEVELOPER_DIR: ${{ matrix.environment.developer-dir }}
+
+    steps:
+      - name: Free disk space by removing .NET, Android and Haskell
+        if: ${{ runner.os == 'Linux' }}
+        run: sudo rm -rf /usr/share/dotnet /usr/local/lib/android /opt/ghc
+
+      - name: Checkout sources
+        uses: actions/checkout@v4
+
+      - name: Restore Boost submodule from cache
+        uses: actions/cache@v4
+        with:
+          path: |
+            3party/boost
+            .git/modules/3party/boost
+          key: boost-submodule
+
+      - name: Parallel submodules checkout
+        run: git submodule update --depth 1 --init --recursive --jobs=20
+
+      - name: Install build tools and dependencies (Linux)
+        if: ${{ runner.os == 'Linux' }}
+        run: |
+          sudo apt update -y
+          sudo apt install -y \
+              ninja-build \
+              libgl1-mesa-dev \
+              libglvnd-dev \
+              libharfbuzz-dev \
+              libxrandr-dev \
+              libxinerama-dev \
+              libxcursor-dev \
+              libxi-dev \
+              qt6-base-dev \
+              libqt6svg6-dev \
+              qt6-positioning-dev \
+              libqt6positioning6-plugins \
+              libqt6positioning6
+
+      - name: Install build tools and dependencies (macOS)
+        if: ${{ runner.os == 'macOS' }}
+        env:
+          HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK: 1
+          HOMEBREW_NO_ANALYTICS: 1
+          HOMEBREW_NO_INSTALL_CLEANUP: 1
+        run: |
+          brew install ninja qt@6
+          pip3 install tornado --break-system-packages
+
+      - name: Configure repository
+        run: ./configure.sh
+
+      - name: Configure ccache
+        uses: hendrikmuhs/ccache-action@v1.2
+        with:
+          key: ${{ github.workflow }}-${{ matrix.environment.os }}-${{ matrix.environment.cc }}-${{ matrix.environment.no-unity }}-${{ matrix.cmake-build-type }}
+
+      - name: Configure cmake
+        env:
+          CC: ${{ matrix.environment.cc }}
+          CXX: ${{ matrix.environment.cxx }}
+          CMAKE_C_COMPILER_LAUNCHER: ccache
+          CMAKE_CXX_COMPILER_LAUNCHER: ccache
+        run: |
+          cmake . -B ${{ env.BUILD_DIR }} -G Ninja \
+            -DCMAKE_BUILD_TYPE=${{ matrix.cmake-build-type }} \
+            -DCMAKE_C_FLAGS=-g1 \
+            -DCMAKE_CXX_FLAGS=-g1 \
+            -DUNITY_DISABLE=${{ matrix.environment.no-unity }}
+
+      - name: Compile
+        working-directory: ${{ env.BUILD_DIR }}
+        run: ninja
+
```

Comment:
nit: Seeing ccache hit rate can be convenient to debug misses
```suggestion

      - name: Print CCache stats
        run: ccache -s
```

Reply:
They are visible here:
https://github.com/organicmaps/organicmaps/actions/runs/15260627592

And in each job:
![image](https://github.com/user-attachments/assets/9c63923e-4ee9-4ad8-b683-25daa74ae190)